src/components/ButtonWithIcon/ButtonsWithIcon.stories.js

```diff
@@ -7,27 +7,36 @@ import { action } from '@storybook/addon-actions';
 import { withInfo } from '@storybook/addon-info';
 import { check, settings } from 'react-icons-kit/feather';
 
+import reactIconSrc from '../../assets/images/react-icon.svg';
+import Showcase from '../Showcase';
+import Paragraph from '../Paragraph';
 import ButtonWithIcon from './ButtonWithIcon';
 
 storiesOf('ButtonWithIcon', module).add(
   'default',
-  withInfo(`
-    Basic usage of Button Component.
-    A list of all icons can be found [here](http://wmira.github.io/react-icons-kit/#/iconset/feather)
-    `)(() => (
+  withInfo()(() => (

```

Comment:
I killed the `withInfo` from here because I realized that a better place for this info is by the `Icon` component story (which doesn't actually exist yet, although I added a note for it to #131)

The only place we're using this component is in the onboarding, and in that case we're not actually using react-icons-kit icons, we're using React/Gatsby logo. So it felt a bit weird to talk about icon selection here, when it's not actually a hard rule to use those icons